### PR TITLE
6957 settings page

### PR DESCRIPTION
**Pull request by asanametove**

### CODE REVIEW

File: conference/react-js/src/constants/actions-types.jsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 const actions = {
-  SHOW_SUCCESS_RESET_PASSWORD_MESSAGE: 'show',
+  SHOW_SUCCESS_RESET_PASSWORD_MESSAGE: 'showBlock',
```
Comment (SupraVlad) [inline conference/react-js/src/constants/actions-types.jsx]: use `show_block` name, please

**Review by SupraVlad — CHANGES_REQUESTED** [review]

File: conference/react-js/src/components/SettingsPage/style.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    &:hover .settings__edit {
+      text-decoration: underline;
+      @media ($mnw: $mdmn) {
+        &:before {
+          content: '';
+          display: block;
+          width: 15px;
+          height: 15px;
+          float: left;
+          //background: url(assets/img/pen-icon.svg) no-repeat center;
```
Comment (YaroslavPerepelychnyi) [inline conference/react-js/src/components/SettingsPage/style.scss]: clean?

Comment (asanametove) [reply conference/react-js/src/components/SettingsPage/style.scss]: I just forgot to include the image to the final version.

File: conference/react-js/src/constants/actions-types.jsx (hunk 1 above)
Comment (asanametove) [reply conference/react-js/src/constants/actions-types.jsx]: We do not know how it appeared here. Maybe it`s IDE refactoring tools(rename). I left only 'show' there.

**Review by SupraVlad — APPROVED** [review]

File: conference/react-js/src/components/NameBrief/index.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const NameBrief = ({ fname, lname, show, title }) => (
+  <div
+    role="button"
+    tabIndex={0}
+    onClick={show}
+    className="settings__brief settings__row"
+  >
+    <div className="settings__title">{title}</div>
+    <div className="settings__row-content">{`${fname} ${lname}`}</div>
```
Comment (blyzniuk) [inline conference/react-js/src/components/NameBrief/index.jsx]: ```>{fname} {lname}</div>```

File: conference/react-js/src/components/SettingsPage/index.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        { error &&
+          <div className="settings__info settings__error">
+            {error}
+          </div> }
+        { message &&
+          <div className="settings__info settings__success">
+            {message}
+          </div> }
+        <SettingsContainer
+          setMessage={msg => this.setState({ message: msg, error: null })}
```
Comment (blyzniuk) [inline conference/react-js/src/components/SettingsPage/index.jsx]: This 3 functions are creating on every render. Make them class methods.

File: conference/react-js/src/components/SlideBlock/index.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SlideBlock = ({ isOpened, children }) => (
+  <div>
+    {
+      isOpened ?
```
Comment (blyzniuk) [inline conference/react-js/src/components/SlideBlock/index.jsx]: It is not necessary, but how about ```children[+isOpened]``` ? :)

File: conference/react-js/src/containers/SignInForm/index.jsx [hunk 6]
```diff
@@ -8,7 +8,7 @@ import { bindActionCreators } from 'redux';
 import loadData from '../../actions/load';
 import { forgotPassword, signUp } from '../../constants/route-url';
 import login from '../../actions/login';
-import getUserInfo from '../../actions/getUserInfo';
+import getUserInfo from '../../actions/get-user-info';
```
Comment (blyzniuk) [inline conference/react-js/src/containers/SignInForm/index.jsx:11]: Aren't you use camel case?

File: conference/react-js/src/components/SlideBlock/index.jsx (hunk 5 above)
Comment (asanametove) [reply conference/react-js/src/components/SlideBlock/index.jsx]: I think it's a good idea.

**Review by YaroslavPerepelychnyi — APPROVED** [review]

File: conference/react-js/src/containers/SignInForm/index.jsx (hunk 6 above)
Comment (asanametove) [reply conference/react-js/src/containers/SignInForm/index.jsx:11]: I think camel case is not good at file names, maybe we need to talk about it...

**Review by blyzniuk — APPROVED** [review]

**Pull request merged**